### PR TITLE
demo(a11y): Add skip navigation link for a11y demo

### PR DESCRIPTION
**Pull request by tinayuangao**

### CODE REVIEW

File: src/demo-app/a11y/a11y.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <h1>Accessibility Demo</h1>
 
 <nav>
+  <a (click)="skipNavigation()" href="javascript:void(0)">Skip navigation</a>
```
Comment (jelbourn) [inline src/demo-app/a11y/a11y.html]: Can this be done with a url fragment? Using `javascript:void(0)` is a pattern that is generally considered harmful to a11y

Comment (tinayuangao) [inline src/demo-app/a11y/a11y.html]: Fragment is not working https://github.com/angular/angular/issues/13636 
We can change this to a material button and remove `href="javascript:void(0)"`

Comment (jelbourn) [inline src/demo-app/a11y/a11y.html]: Chatted w/ Jason a bit about this to see if we could do any kind of workaround. And... not really.

Does it work if you make the `href="#maincontent"` and still use the click handler to do the work?

Comment (tinayuangao) [inline src/demo-app/a11y/a11y.html]: Changed to use `routerLink` with current path and it works. Removed `javascript:void(0)`

**Comment by kara** [comment]
@tinayuangao Can you check out the linter error? https://travis-ci.org/angular/material2/jobs/264586210

**Pull request merged**